src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  activeTodosTotal: number,
+  isLeftActiveTodos: boolean,
+  filterValue: string,
+  setFilterValue: (value: string) => void,
+  completedTodosIds: number[],
+  onDelete: (id: number[]) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  activeTodosTotal,
+  isLeftActiveTodos,
+  filterValue,
+  setFilterValue,
+  completedTodosIds,
+  onDelete,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${activeTodosTotal} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={classNames(
+            'filter__link',
+            { selected: filterValue === 'all' },
+          )}
+          onClick={() => setFilterValue('all')}
+        >
+          All
+        </a>
+
+        <a
+          data-cy="FilterLinkActive"
+          href="#/active"
+          className={classNames(
+            'filter__link',
+            { selected: filterValue === 'active' },
+          )}
+          onClick={() => setFilterValue('active')}
+        >
+          Active
+        </a>
+        <a
+          data-cy="FilterLinkCompleted"
+          href="#/completed"
+          className={classNames(
+            'filter__link',
+            { selected: filterValue === 'completed' },
+          )}
+          onClick={() => setFilterValue('completed')}
+        >
+          Completed
+        </a>
+      </nav>
+
+      <button
+        data-cy="ClearCompletedButton"
+        type="button"
+        className="todoapp__clear-completed"
+        onClick={() => onDelete(completedTodosIds)}
+      >
+        {!isLeftActiveTodos && 'Clear completed'}
+      </button>
```

Comment:
the empty button on page if I don’t have completed todos 🥲
how it looks on your page:
<img width="710" alt="Screenshot 2022-10-05 at 00 48 06" src="https://user-images.githubusercontent.com/47606379/193936586-7d151a13-7f89-49cb-904a-ef62029edc4a.png">


how it looks on [mate demo page](https://mate-academy.github.io/react_todo-app-with-api/):
<img width="748" alt="Screenshot 2022-10-05 at 00 48 37" src="https://user-images.githubusercontent.com/47606379/193936656-6ded4070-339a-4b19-abf9-a48c73d566c3.png">

Reply:
Yes) I've already fixed it in the next part. Just forgot to fix it here))